ceph/rados/rados_scrub.py

```diff
@@ -0,0 +1,159 @@
+"""
+   This module contains the methods required for scrubbing.
+
+   1.To set the parameters for scrubbing initially required the
+     cluster time and day details.get_cluster_date method provides
+     the details
+
+   2.set_osd_configuration method  used to set the configuration
+     parameters on the cluster.
+
+   3.get_osd_configuration  method is used to get the configured parameters
+     on the cluster.
+
+     NOTE: With set_osd_configuration & get_osd_configuration methods can
+          use to set the get the any OSD configuration parameters.
+
+   4. get_pg_dump  method is used to get the pg dump details from the cluster
+
+   5. verify_scrub  method used for the verification of scheduled scrub
+      happened or not.
+"""
+
+import logging
+from collections import defaultdict
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+
+log = logging.getLogger(__name__)
+
+
+class RadosScrubber:
+    def __init__(self, node: CephAdmin):
+        """
+        initializes the env to run rados commands
+        Args:
+            node: CephAdmin object
+        """
+        self.node = node
+        self.ceph_cluster = node.cluster
+        self.rados_obj = RadosOrchestrator(node=node)
+
+    def run_container_cmd(self, cmd: str):
```

Comment:
IMO, this method would need to be part of `RadosOrchestrator`

Reply:
I believe that all/most of the commands executed in 5.x, by default, uses method present with ` CephAdmin `, which by default executes command inside the ` cephadm ` Container itself. 

Even ` RadosOrchestrator ` class makes use of ` CephAdmin ` object for command execution. 

We can make the container calls directly, not sure if another method would be necessary to do the same.

Reply:
deleted the method called shell command